test/include/test/test.h

```diff
@@ -1,4 +1,4 @@
-#define ASSERT_BUFSIZE	256
+#define ASSERT_BUFSIZE	4096
```

Comment:
Is this change intentional?

Reply:
Kind of. Without this change not all output will fit in the buffer to print assert message, when test fails. But this is not necessary for the primary change. Let me try to back it out and check if Windows CI will recover.

Reply:
@interwq, you were right, that's it. I suspect it is likely because internal `MALLOC_PRINTF_BUFSIZE` is 4096 bytes as well and they might interact in non-obvious way. I'll try to understand it better if I'll have time, meanwhile remove this change from PR.